wger/nutrition/migrations/0002_auto_20180430_1302.py

```diff
@@ -0,0 +1,30 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.9.13 on 2018-04-30 10:02
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('nutrition', '0001_initial'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='ingredient',
+            name='status',
+            field=models.CharField(choices=[('1', 'Pending'), ('2', 'Accepted'), ('3', 'Declined'), ('4', 'Submitted by administrator'), ('5', 'System ingredient')], default='1', editable=False, max_length=2),
+        ),
+        migrations.AlterField(
+            model_name='meal',
+            name='order',
+            field=models.IntegerField(blank=True, editable=False, verbose_name='Order'),
+        ),
+        migrations.AlterField(
+            model_name='mealitem',
+            name='order',
+            field=models.IntegerField(blank=True, editable=False, verbose_name='Order'),
+        ),
+    ]
```

Comment:
Great job implementing your task. What do you think about adding tests for the added functionality?

Reply:
there are already existing tests on the same